lib/include/tick/survival/model_sccs.h

```diff
@@ -96,6 +97,61 @@ class DLL_PUBLIC ModelSCCS : public ModelLipschitz {
       out[i] = exp(x[i] - x_max) / sum;  // overflow-proof
     }
   }
+
+  template <class Archive>
+  void serialize(Archive &ar) {
```

Comment:
Why this method as you are already implementing load and save?

Reply:
typo ^_^